examples/00_getting_started.py

```diff
@@ -12,6 +12,13 @@
 
 dh = ru.DebugHelper()
 
+N = 2
+N = 1024 * 64
+N = 1024 * 2
+
+P = 16
+U = 33
+
```

Comment:
What are these and where are they being used?

Reply:
For completeness: the debug helper attaches some hooks which dumps stack traces for *all* threads on termination.

Reply:
okay!